packages/providers/fal/avatar-generation/fal_avatar/models/kling.py

```
@@ -0,0 +1,480 @@
+
+            return AvatarGenerationResult(
+                success=True,
+                video_url=result["video"]["url"],
+                duration=video_duration,
+                cost=self.estimate_cost(video_duration),
+                processing_time=response["processing_time"],
+                model_used=self.model_name,
+                metadata={"aspect_ratio": arguments["aspect_ratio"]},
+            )
+
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)